src/modules/vtol_att_control/tailsitter.cpp

```diff
@@ -131,6 +131,10 @@ void Tailsitter::update_vtol_state()
 	 * For the backtransition the pitch is controlled in MC mode again and switches to full MC control reaching the sufficient pitch angle.
 	*/
 
+	matrix::Quaternion<float> q(&_v_att->q[0]);
+	matrix::Euler<float> euler(q);
+	float pitch = euler(1);
```

Comment:
If this is just another name for `euler(1)`, what about either:

`const float pitch = euler(1);`
or:
`float &pitch = euler(1);`